### ISSUE TITLE
search broken

### ISSUE THREAD

**Issue by tokisuno** [manuelvargastapia/libgen_mobile_app#38]
has been for some time.

**Comment by ghost** [comment]
yeah, seems to be dead, which is a shame since the UI is very good-looking and responsive. i hope someone picks it up and re-wires the server - I wonder if the libgen team itself could be interested in something like this

**Comment by tokisuno** [comment]
i wish. it would eliminate the step of downloading on my pc and transfering books through kde connect.
